services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_public.py

```diff
@@ -22,6 +24,9 @@ class DynamicSidecarClient:
     def __init__(self, app: FastAPI):
         self.thin_client: ThinDynamicSidecarClient = ThinDynamicSidecarClient(app)
 
+    def get_async_client(self) -> AsyncClient:
+        return self.thin_client.get_async_client()
```

Comment:
isn't that defeating the purpose of having a client if you pass the internals out??

Reply:
I've refactored this and is no longer public.